src/conditional-flow/multiple-conditions.js

```diff
@@ -12,7 +12,7 @@ function isInRange (num, lower, upper) {
 // if statement.
 function isHelloOrGoodbye (val1) {
 
-  // TODO: write code in this function body to pass the tests
+return val1==="Hello" || val1==="Goodbye";
```

Comment:
Great use of the `===` here. You don't know if val1 is a string, and by using the `===` you're forcing javascript to check the values as if they were a string